ui/src/pages/DiscoverEvents.jsx

```diff
@@ -15,7 +15,20 @@ class DiscoverEvents extends React.Component{
     this.props.getEvents();
   }
 
+  getDefaultView() {
```

Comment:
We decided to change the default view to `<><Header /><div className="container" /></>` to avoid the annoying flash

Reply:
See: https://github.com/thenathanyang/tripify/commit/5c22755fff03427fe78a2b3acc54fca17504e109#diff-e874fe9d4cf6160413cbb377a6655629R26